pkg/volume/glusterfs/glusterfs.go

```diff
@@ -319,10 +319,11 @@ func (b *glusterfsMounter) setUpAtInternal(dir string) error {
 		// its own log based on PV + Pod
 		log = path.Join(p, b.pod.Name+"-glusterfs.log")
 
+		// Use derived/provided log file in gluster fuse mount
```

Comment:
should remove the word "provided" since it's only derived now

Reply:
Done.. thanks!